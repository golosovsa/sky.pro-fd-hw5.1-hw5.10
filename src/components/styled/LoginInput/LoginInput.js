import styled from "styled-components";
import { applyThemeMixin } from "../../../globals/theme/index";
import { layoutTextMixin, calcHeight } from "../../../globals/layout/index";

const LoginInput = styled.input`
    border: none;
    ${applyThemeMixin("login", "input")}
    ${layoutTextMixin(18, 24, 400)}
    font-feature-settings: 'pnum' on, 'lnum' on;
    border-bottom-width: ${calcHeight(1)};
    border-bottom-style: solid;
    padding: ${calcHeight(8)} 0;
    align-self: stretch;
    outline: none;
`;

export default LoginInput;
