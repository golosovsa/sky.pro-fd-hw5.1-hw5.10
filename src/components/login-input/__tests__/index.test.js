import ShallowRenderer from 'react-test-renderer/shallow'
import LoginInput from '..'

const LoginInputWrapped = () => {
  return <LoginInput value={123} placeholder={123} />
}

describe('<LoginInput />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<LoginInputWrapped />)

      expect(tree).toMatchSnapshot()
    })
  })
})
