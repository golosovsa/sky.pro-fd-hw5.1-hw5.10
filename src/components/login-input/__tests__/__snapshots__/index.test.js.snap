// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginInput /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.input
  placeholder={123}
  value={123}
/>
`;
